### PR TITLE
Conplete html-syntax.

### PR DESCRIPTION
**Pull request by jayb0c**
Screen Shot 2021-06-21 at 9 34 14 PM

### CODE REVIEW

File: html-syntax/index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Syntax</title>
+  </head>
+
+  <body>
+    <img src="https://assets.pokemon.com/assets/cms2/img/pokedex/full/025.png">
```
Comment (BrettAlbright) [inline html-syntax/index.html]: Make sure to add the `alt` attribute with a suitable description of the image.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
one tiny update!

File: html-syntax/index.html (hunk 1 above)
Comment (jayb0c) [reply html-syntax/index.html]: Thanks so much! I got the bot message at work and went, "uh oh the alt". I'll fix it now

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**